.github/workflows/ci.yml

```diff
@@ -5,16 +5,20 @@ on: [push, pull_request]
 jobs:
   build:
     runs-on: ubuntu-latest
+    services:
+      trino:
+        image: trinodb/trino:355
+        ports:
+          - 8080:8080
     strategy:
       fail-fast: false
       matrix:
-        python-version: [
-          3.5,
-          3.6,
-          3.7,
-          3.8,
-          3.9,
-        ]
+        python-version:
+          - 3.5
```

Comment:
We removed support for 3.5 in #83. Could you rebase on upstream to resolve the conflict?